package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "setup-packer",
-  "version": "2.1.0",
+  "version": "3.0.0",
```

Comment:
Can you undo the version change for now, I'll want to update https://github.com/hashicorp/github-actions-core/tree/main as well to use node 20 and node 20 version of the @types library so that we use node 20 consistently across the action.